### DISCUSSION TITLE
just display the pdf not save to disk?

### DISCUSSION THREAD

**Discussion by salassa** [QuestPDF/QuestPDF#444] [Ideas]
can i just display pdf data and not have to or without saved/generate it to the disk, thanks

**Comment by girlpunk** [comment]
There are overloads for `GeneratePdf` that allow outputting to a stream or byte array, those should do what you want.

**Reply by salassa** [reply]
Thank you very much @girlpunk

**Comment by DawidRobertNowak** [comment]
Hello,

We are closing this thread due to inactivity. Thank you for taking the time to reach out and contribute to the project — your feedback helps us improve QuestPDF.

Should you encounter any further problems, please don't hesitate to open a new issue.

Thank you!
